packages/network/test/integration/tracker-instructions.test.ts

```diff
@@ -91,16 +98,18 @@ describe('check tracker, nodes and statuses from nodes', () => {
             waitForEvent(node2, NodeEvent.NODE_SUBSCRIBED)
         ])
 
+        /*
```

Comment:
Cannot get the portion:
`waitForEvent(tracker.trackerServer, TrackerServerEvent.NODE_STATUS_RECEIVED)`
of code to run properly, always gets timed out and can't figure out why